packages/devtools_app/lib/src/shared/preferences/_inspector_preferences.dart

```diff
@@ -30,7 +30,9 @@ class InspectorPreferencesController extends DisposableController
       serviceConnection.inspectorService;
 
   final _hoverEvalMode = ValueNotifier<bool>(false);
-  final _inspectorV2Enabled = ValueNotifier<bool>(false);
+  final _legacyInspectorEnabled = ValueNotifier<bool>(false);
+  // TODO(https://github.com/flutter/devtools/issues/1423): Default to true
+  // after verifying auto-refreshes are performant.
   final _autoRefreshEnabled = ValueNotifier<bool>(true);
```

Comment:
do we still need this todo? It looks like we are already defaulting to true

Reply:
Ah forgot to delete that TODO when I switched auto-refresh on. I've removed it.